.yarn/patches/@concordium-web-sdk-npm-7.3.2-42c2804ca6.patch

```diff
@@ -0,0 +1,130 @@
++</module>
+\ No newline at end of file
+diff --git a/.idea/workspace.xml b/.idea/workspace.xml
+new file mode 100644
+index 0000000000000000000000000000000000000000..097b6eae9ea7a26f0c80bee93d9c7ef474fbbe1a
+--- /dev/null
++++ b/.idea/workspace.xml
+@@ -0,0 +1,49 @@
++<?xml version="1.0" encoding="UTF-8"?>
++<project version="4">
++  <component name="ChangeListManager">
++    <list default="true" id="f201ab91-21b9-4629-8344-69fbd5d236e8" name="Changes" comment="" />
++    <option name="SHOW_DIALOG" value="false" />
++    <option name="HIGHLIGHT_CONFLICTS" value="true" />
++    <option name="HIGHLIGHT_NON_ACTIVE_CHANGELIST" value="false" />
++    <option name="LAST_RESOLUTION" value="IGNORE" />
++  </component>
++  <component name="MarkdownSettingsMigration">
++    <option name="stateVersion" value="1" />
++  </component>
++  <component name="ProjectColorInfo"><![CDATA[{
++  "associatedIndex": 1
++}]]></component>
++  <component name="ProjectId" id="2eY2L3aOsQWbvovINdrNOBtrtco" />
++  <component name="ProjectViewState">
++    <option name="autoscrollFromSource" value="true" />
++    <option name="hideEmptyMiddlePackages" value="true" />
++    <option name="showLibraryContents" value="true" />
++  </component>
++  <component name="PropertiesComponent"><![CDATA[{
++  "keyToString": {
++    "RunOnceActivity.OpenProjectViewOnStart": "true",
++    "RunOnceActivity.ShowReadmeOnStart": "true",
++    "last_opened_file_path": "/tmp/xfs-3f59d0d4/user",
++    "node.js.detected.package.eslint": "true",
++    "node.js.selected.package.eslint": "(autodetect)",
++    "nodejs_package_manager_path": "npm",
++    "settings.editor.selected.configurable": "vcs.Git",
++    "vue.rearranger.settings.migration": "true"
++  }
++}]]></component>
++  <component name="SpellCheckerSettings" RuntimeDictionaries="0" Folders="0" CustomDictionaries="0" DefaultDictionary="application-level" UseSingleDictionary="true" transferred="true" />
++  <component name="TaskManager">
++    <task active="true" id="Default" summary="Default task">
++      <changelist id="f201ab91-21b9-4629-8344-69fbd5d236e8" name="Changes" comment="" />
++      <created>1712067131693</created>
++      <option name="number" value="Default" />
++      <option name="presentableId" value="Default" />
++      <updated>1712067131693</updated>
++      <workItem from="1712067132751" duration="29000" />
++    </task>
++    <servers />
++  </component>
++  <component name="TypeScriptGeneratedFilesManager">
++    <option name="version" value="3" />
++  </component>
++</project>
+\ No newline at end of file
+diff --git a/package.json b/package.json
+index 2653ea6fbf48c58e34e4b8235fc624d2a358b2e0..e4cbc990fa038fb69504ef7cb28eb83c82178c2b 100644
+--- a/package.json
++++ b/package.json
+@@ -38,6 +38,11 @@
+             "react-native": null,
+             "default": "./lib/esm/pub/wasm.js"
+         },
++        "./types/*": {
++            "types": "./lib/esm/types/*.d.ts",
++            "bun": "./src/pub/*.ts",
```

Comment:
**Suggestion:** I don't think "bun" is right.
It doesn't really matter for this, since we don't use it here, but we probably want this in the JS SDK, where it needs to be correct.

```suggestion
+            "bun": "./src/types/*.ts",
```